sdk/purview/azure-purview-sharing/azure/purview/sharing/aio/operations/_operations.py

```diff
@@ -1205,22 +1206,14 @@ def get_long_running_output(pipeline_response):
 
     @distributed_trace
     def list_attached(
-        self,
-        *,
-        reference_name: str,
-        skip_token: Optional[str] = None,
-        filter: Optional[str] = None,
-        orderby: Optional[str] = None,
-        **kwargs: Any
+        self, *, reference_name: str, filter: Optional[str] = None, orderby: Optional[str] = None, **kwargs: Any
     ) -> AsyncIterable[JSON]:
         """Get a list of attached received shares.
 
         List attached received shares.
 
         :keyword reference_name: A name that references a data store. Required.
         :paramtype reference_name: str
-        :keyword skip_token: The continuation token to list the next page. Default value is None.
```

Comment:
Is this a by design breaking change?

Reply:
Yes this is by design and was recommended by the Open API Specs team: https://github.com/Azure/azure-rest-api-specs/pull/23993#discussion_r1194514618

Reply:
Thanks for the confirmation.

Let's add it into the changelog. :)

Reply:
Under a "Breaking Changes" section? And would saying something like "Changes to the _operations.py file recommended by the Open API Specs team" be good enough? Could you provide clarification on what this breaks?

Reply:
I would cite the actual surface area changed on the SDK, instead of how we arrived at the decision.  The `skip_token` parameter was removed from list methods in favor of placing it within the `nextLink`.  Casing and wording here might need adjusted.

Reply:
It looks like the change is `skip_token` argument is not supported any more.

Given we have not shipped any GA releases yet, we can just put it under other changes or a breaking change with statement like

https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/CHANGELOG.md#breaking-changes-1

Either way should work.

Reply:
Okay! I'll type it up and adjust the wording as needed